src/main/java/org/apache/maven/plugins/gpg/AbstractGpgMojo.java

```diff
@@ -131,13 +131,14 @@ public abstract class AbstractGpgMojo extends AbstractMojo {
      * Server id to lookup the passphrase under Maven settings. <em>Do not use this parameter, it leaks
      * sensitive data. Passphrase should be provided only via gpg-agent or via env variable.
      * If parameter {@link #bestPractices} set to {@code true}, plugin fails when this parameter is configured.</em>
+     * Is programatically defaulted to {@link #GPG_PASSPHRASE}.
      *
      * @since 1.6
      * @deprecated Do not use this configuration, it may leak sensitive information. Rely on gpg-agent or env
      * variables instead.
      **/
     @Deprecated
-    @Parameter(property = "gpg.passphraseServerId", defaultValue = GPG_PASSPHRASE)
+    @Parameter(property = "gpg.passphraseServerId")
```

Comment:
I would like to not remove default values .... can be assumed as present by many configurations